src/core.js

```diff
@@ -510,6 +509,15 @@ $.extend( $.validator, {
 			return this.size() === 0;
 		},
 
+		// Check if the given element is valid
+		// return
+		//          true  If the field is valid
+		//         false  If the field is invalid
+		//     undefined  If the field is not validated yet.
+		isValidElement: function( element ) {
```

Comment:
I added also this function in order to check if the given element is valid or not. I don't know if it can be in use to others, but I used it in this patch.